Description: Build shared lib with bioconda's htslib, tabixpp, and libssw
 The upstream repo includes htslib as a submodule that gets
 built with the tabixpp code. This patch changes this to use
 bioconda's shared libhts from the archive.
 Also builds a shared library instead of a static one.
Author: Andreas Tille <tille@debian.org> modified for bioconda by Jon Puritz jpuritz@gmail.com
--- vcflib/CMakeLists.txt
+++ vcflib/CMakeLists.txt
@@ -86,14 +86,13 @@
     src/pdflib.cpp
     src/cdflib.cpp
     src/split.cpp
-    src/ssw.hpp
-    src/ssw_cpp.hpp
     filevercmp/filevercmp.c
     )
 
 target_link_libraries(vcflib
     PkgConfig::SMITHWATERMAN
     PkgConfig::FASTAHACK
+    ssw
     )
 
 set(BINS
--- vcflib/src/Variant.h
+++ vcflib/src/Variant.h
@@ -28,7 +28,7 @@
 #include "join.h"
 #include <tabix.hpp>
 #include <smithwaterman/SmithWatermanGotoh.h>
-#include "ssw_cpp.hpp"
+#include <ssw_cpp.h>
 #include "convert.h"
 #include "multichoose.h"
 #include <fastahack/Fasta.h>
